guide/inheritance-wallet/introduction.md

```diff
@@ -0,0 +1,132 @@
+   image = "/assets/images/guide/inheritance-wallet/introduction/family-overview.png"
+   retina = "/assets/images/guide/inheritance-wallet/introduction/family-overview.png"
+   mobile = "/assets/images/guide/inheritance-wallet/introduction/family-overview.png"
+   mobileRetina = "/assets/images/guide/inheritance-wallet/introduction/family-overview.png"
+   alt-text = "An illustration showing the Jones family."
+   width = 1600
+   height = 800
+%}
+
+Alice and Bob have been using a shared multi-key wallet as a family checking account for a few years already. This wallet is used for general spending, while each of them manage their income, personal spending and savings using their individual single-key wallets. 
+
+The couple now feels that the time is right to consolidate the family savings and to include their children in securing them. They will still keep the savings separated from their checking wallet.
+
+## User requirements
+The couple has been thinking about what a good solution would look like for them. Here is summary of the most important aspects:
+
+1. **Privacy:** While they are alive and well, they don’t want Christina and David to be able to access the funds. The children should not even know how much they have or see their transactions.
+2. **Flexibility:** The wallet configuration needs to be relatively easy to change, in case their life circumstances change.
+3. **Interoperability:** The wallet itself should be interoperable with other applications, so that it can be easily imported into another wallet application, in case there is a critical bug or the project is no longer supported by the developers.
+4. **Legal support:** Alice and Bob also want to involve Edward, their lawyer, to assist in case there are any problems between the children or their families. He will also be able to work with the legal inheritance processes in their jurisdiction.
+
+## Wallet configuration
+Based on their initial requirements, Alice and Bob decided on having two key sets. One primary key set that they will use in times of normal operations. In addition, they want to create an inheritance key set that unlocks after a certain time. Their children and Edward will each hold one these keys.  
+
+{% include picture.html
+   image = "/assets/images/guide/inheritance-wallet/introduction/key-scheme.png"
+   retina = "/assets/images/guide/inheritance-wallet/introduction/key-scheme.png"
+   mobile = "/assets/images/guide/inheritance-wallet/introduction/key-scheme.png"
+   mobileRetina = "/assets/images/guide/inheritance-wallet/introduction/key-scheme.png"
+   alt-text = "An illustration showing that Alice and Bob control three keys, while Christina, David and Edward control one key each."
+   width = 1600
+   height = 800
+%}
+
+Below is an overview of what the configuration of the family's savings wallet will look like:
+
+- **A 2-of-3 primary key set for normal spending**. Since it is a savings wallet, Alice and Bob will spend only very infrequently.
```

Comment:
Good question. My plan was to not touch the topic in too much detail. It's already a pretty long page. 

That being said. My idea was that each of them store their own key independently. The third key is stored in a shared bank deposit box.

Reply:
Agree. But this is a decision that every family will have to take on their own, based on their specific situation. Plus, the recovery path even lowers the quorum after 6 months. 

Of course, if the relationship between the parents is not a very good one, they might go a different route. They probably would not have a shared savings wallet in the first place, but keep their savings separate, just like before. They also would not have a recovery path set up.

It all comes down to personal choice and what the perceived risk factors are for the users. In this case, the main risk is perceived as self-inflicted loss. The goal of the reference design is to describe the UX.

That being said, it could be a good idea to create a child page under the [Bitcoin backups](https://bitcoin.design/guide/how-it-works/backups/) page, where we could describe a couple of scenarios in more detail 🤔.